fiftyone/server/view.py

```diff
@@ -626,12 +626,19 @@ def _make_keypoint_list_filter(args, view, path, field):
 
     if isinstance(field.field, (fof.FloatField, fof.IntField)):
         f = F(name)
-        mn, mx = args["range"]
-        expr = (f >= mn) & (f <= mx)
-        if args["exclude"]:
-            expr = ~expr
+        #adding support if filter will applied on categorical variable i.e. 'labels' 
```

Comment:
Thanks for catching this. Is 'labels' field intField or stringField?

When the field.field is FloatField or IntField, it will be rendered with the `numericFilter`as a range slide rather than the categorical filter. We can add a separate function to handle StringField. 

I would suggest:
```
    if isinstance(field.field, (fof.FloatField, fof.IntField)):
        f = F(name)
        mn, mx = args["range"]
        expr = (f >= mn) & (f <= mx)
        if args["exclude"]:
            expr = ~expr

        return {"filter": expr}
    
    if isinstance(field.field, (fof.StringField)):
        f = F(name)
        if 'values' in args:
            val = args["values"]
            expr = f.is_in(val)
            if args["exclude"]:
                expr = ~expr
            return {"filter": expr}
```

Reply:
Yes, it was for string fields . I have updated accordingly.